buku

```diff
@@ -4816,6 +4823,33 @@ def prompt(obj, results, noninteractive=False, deep=False, listtags=False, sugge
             show_taglist(bdb)
             continue
 
+        if (nav+' ').startswith('DB '):
+            dbpath, dbfile = os.path.split(bdb.dbfile)
+            if nav == 'DB':
+                print(f'Available DB files (in {dbpath}):')
+                for s in sorted(s for s in os.listdir(dbpath) if s.endswith('.db')):
+                    print(('*' if s == dbfile else ' '), s.removesuffix('.db'))
```

Comment:
If used without argument, `DB` command prints out current location and the sorted list of `.db` files in it. The current DB file is marked in the output.